packages/ingest/granule.js

```diff
@@ -764,43 +741,55 @@ async function moveGranuleFile(source, target, options) {
 /**
  * move granule files from one s3 location to another
  *
- * @param {Object[]} sourceFiles - array of file objects
- * @param {string} sourceFiles[].name - file name
- * @param {string} sourceFiles[].bucket - current bucket of file
- * @param {string} sourceFiles[].filepath - current bucket location of file
+ * @param {string} granuleId - granuleiId
+ * @param {Array<Object>} sourceFiles - array of file objects
+ * @param {string} sourceFiles.name - file name
+ * @param {string} sourceFiles.bucket - current bucket of file
+ * @param {string} sourceFiles.filepath - current s3 key of file
  * @param {Object[]} destinations - array of objects defining the destination of granule files
- * @param {string} destinations[].regex - regex for matching filepath of file to new destination
- * @param {string} destinations[].bucket - aws bucket
- * @param {string} destinations[].key - filepath on the bucket for the destination
+ * @param {string} destinations.regex - regex for matching filepath of file to new destination
+ * @param {string} destinations.bucket - aws bucket of the destination
+ * @param {string} destinations.filepath - file path/directory on the bucket for the destination
  * @param {string} distEndpoint - distribution enpoint from config
+ * @param {boolean} published - indicate if publish is needed
  * @returns {Promise<undefined>} returns `undefined` when all the files are moved
  */
-async function moveGranuleFiles(sourceFiles, destinations, distEndpoint) {
+async function moveGranuleFiles(granuleId, sourceFiles, destinations, distEndpoint, published) {
```

Comment:
It seems like we don't really need to return anything here, since we're returning either {} or Promise.resolve